var/spack/repos/builtin/packages/intltool/package.py

```diff
@@ -36,7 +36,7 @@ class Intltool(AutotoolsPackage):
     version('0.51.0', '12e517cac2b57a0121cda351570f1e63')
 
     # requires XML::Parser perl module
-    # depends_on('perl@5.8.1:', type='build')
+    depends_on('perl-xml-parser', type=('build', 'run'))
```

Comment:
> In python extensions, python is not mentioned as dependency, is it because is the PythonPackage class that takes care of?

Yes.

> could it be related to not explicitly mention all the possible X11 dependencies in turbovnc?

That could very well be the case, although I don't know enough about turbovnc to confirm.